package.json

```diff
@@ -10,7 +10,7 @@
     "start": "yarn workspace nucleus start",
     "test": "lerna run test --no-private --stream --concurrency 1",
     "build": "lerna run build --parallel --no-private",
-    "lint:js": "eslint . --fix --ignore-path .gitignore",
```

Comment:
As we are no more using .gitignore for skipping lint files, we are removing this line. This leads to falling back to default '.eslintignore' for skipping files.